app/src/main/java/fr/gaulupeau/apps/Poche/ui/ReadArticleActivity.java

```diff
@@ -332,7 +346,7 @@ public boolean onOptionsItemSelected(MenuItem item) {
 
     @Override
     public boolean dispatchKeyEvent(KeyEvent event) {
-        if (event.getAction() == KeyEvent.ACTION_DOWN) {
+        if (event.getAction() == KeyEvent.ACTION_UP) {
```

Comment:
Partially breaks scrolling with volume buttons: the `ACTION_DOWN` event is not consumed by the app.

Reply:
my latest commit should have fixed this issue, thanks for pointing it out.